File: Days/Javascript/day2.js
console.log('Compute multiplication for arr: 1, 2, 3, 4, 5');

let arr = [1, 2, 3, 4, 5];

let internalArr = [];

for(var i = 0; i < arr.length; i++){
    var product = 1;
    for(var j = 0; j < arr.length; j++){
        if(j != i){
            product = product * arr[j];
        }
    }

    internalArr.push(product);
}

internalArr.forEach(element => {
    console.log(element + " ");    
});

File: Days/Days/Day2/Day2.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Days.Day2
{
    public class Day2
    {
        public void MultiplyArrElementsSkippingCurrentElement()
        {
            var array = new List<int> {1, 2, 3, 4, 5};

            Console.WriteLine("Computing multiplication for array: 1, 2, 3, 4, 5");
            var internalArr = new List<int>();

            foreach (var i in array)
            {
                var product = array.Where(j => j != i).Aggregate(1, (current, j) => current * j);

                internalArr.Add(product);
            }

            foreach (var tem in internalArr)
            {
                Console.Write(tem+" ");
            }
        }
    }
}
